packages/foundry-sdk-generator/src/generate/betaClient/minifyBundle.ts

```diff
@@ -35,13 +38,25 @@ export class ProjectMinifier {
 
   constructor(
     private project: Project,
-    private startingImportSet: Set<string>,
-    private startingFilePath: string,
+    startingImportSet: Set<string>,
```

Comment:
Old code assumed that all the imports you wanted to keep came from a single file. This is not the case since we did legacy-client and so this code changes to take a starting set of `${file}:${importName}` for the tree shaking.